JS/Lesson15/onur/script.js

```diff
@@ -121,10 +57,44 @@ checkIfStringIsUpperCase('write a program that takes in');
 
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
 
+function isNumberPositiveOrNegative(num) {
+	if (num > 0) {
+		console.log('Number is positive');
+	} else if (num < 0) {
+		console.log('Number is negative');
+	} else {
+		console.log('Number is equel to zero');
+	}
+}
+isNumberPositiveOrNegative(65);
+isNumberPositiveOrNegative(-27);
+isNumberPositiveOrNegative(0);
+
 // 6. Write a program that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
 
+function isLeapYear(yearNumber) {
+	if (yearNumber % 4 === 0 && (yearNumber % 100 !== 0 || yearNumber % 400 === 0)) {
```

Comment:
Your solution works (as numbers divisible by 400 are also divisible by 4) but is harder to comprehend. I'd do like this instead: 
```suggestion
	if (yearNumber % 4 === 0 && yearNumber % 100 !== 0 ) || yearNumber % 400 === 0)) {
```
so it is either yearNumber divisible by 4 and not divisible by 100 OR yearNumber divisible by 400.